src/controller/python/ADVANCED_USAGE.md

```diff
@@ -0,0 +1,234 @@
+# Python CHIP Device Controller - Advanced usage
+
+## BLE Virtualization on Linux (Optional)
+
+If you would like to setup virtual BLE central and peripheral in the same
+machine, then follow below steps:
+
+```
+cd third_party/bluez/repo
+./bootstrap
+third_party/bluez/repo/configure --prefix=/usr --mandir=/usr/share/man --sysconfdir=/etc --localstatedir=/var --enable-experimental --with-systemdsystemunitdir=/lib/systemd/system --with-systemduserunitdir=/usr/lib/systemd --enable-deprecated --enable-testing --enable-tools
+make
+```
+
+Note: You also need to install several packages on RPi if you want to build
```

Comment:
This should probably be moved before the bluez build step. Also, there should be no harm in running the command on Ubuntu.